### PR TITLE
RM-9377 Update CoreForms to 8.0.3

### PR DESCRIPTION
**Pull request by a-ctor**
https://re-motion.atlassian.net/browse/RM-9377

### CODE REVIEW

File: Remotion/Web/Core/ExecutionEngine/WxeFunctionStateManager.cs [hunk 1]
```diff
@@ -102,12 +102,14 @@ public WxeFunctionStateManager (HttpSessionStateBase session)
       ArgumentUtility.CheckNotNull("session", session);
       _session = session;
 
-      _functionStates = (Dictionary<string, WxeFunctionStateMetaData>)_session[s_sessionKeyForFunctionStates];
```
Comment (Frazerus) [inline Remotion/Web/Core/ExecutionEngine/WxeFunctionStateManager.cs:105]: I assume these changes are due to the upgrade in coreforms?

**Review by Frazerus — APPROVED** [review]

File: Remotion/Web/Core/ExecutionEngine/WxeFunctionStateManager.cs (hunk 1 above)
Comment (a-ctor) [reply Remotion/Web/Core/ExecutionEngine/WxeFunctionStateManager.cs:105]: Yeah the nullability of the session indexer changed

File: Remotion/Web/Core/ExecutionEngine/WxeFunctionStateManager.cs [hunk 2]
```diff
@@ -172,7 +174,7 @@ public WxeFunctionState GetItem (string functionToken)
       WxeFunctionState functionState;
       lock (_lockObject)
       {
-        functionState = (WxeFunctionState)_session[GetSessionKeyForFunctionState(functionToken)];
+        functionState = (WxeFunctionState)_session[GetSessionKeyForFunctionState(functionToken)]!;
```
Comment (a-ctor) [inline Remotion/Web/Core/ExecutionEngine/WxeFunctionStateManager.cs:177]: @MichaelKetting This is the cheap fix for the nullability problem but we could also backport https://re-motion.atlassian.net/browse/RM-9089 if you want. I don't know if we want that though, so for now its just the nullability override.

Comment (MichaelKetting) [reply Remotion/Web/Core/ExecutionEngine/WxeFunctionStateManager.cs:177]: Cheap fix it is

**Review by MichaelKetting — APPROVED** [review]

**Pull request merged**